recipes/bytecode/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "bytecode" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f78a312880173f1d20aef2eceb8eb959dc6c2167966b757fef201b8af839f802
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - aenum  # [py<36]
```

Comment:
Please try removing the selector, `# [py<36]`.

Reply:
aenum is not required on python 3.6 and above which is why I used the selector. Should we install it unconditionally even if it is not used ?

Reply:
Ah, I see. Yeah, if it is not needed at all, you can remove it. I think CF builds against Python >=3.6 anyway.

Reply:
I removed aeunm and in consequence bumped the minimum python version to 3.6. I could not confirm that CF only build on python 3.6 and above so it sounds safer to make sure we do not generate a bad package.